src/bun.js/bindings/bindings.cpp

```diff
@@ -4326,14 +4326,15 @@ static void fromErrorInstance(ZigException* except, JSC::JSGlobalObject* global,
                 iterator.forEachFrame([&](const V8StackTraceIterator::StackFrame& frame, bool& stop) -> void {
                     ASSERT(except->stack.frames_len < frame_count);
                     auto& current = except->stack.frames_ptr[except->stack.frames_len];
+                    current = {};
```

Comment:
this one should've already be zero'd

Reply:
i cant find a reference for where this is done. and i dont think it hurts. not all fields are being assigned below